api/devices/schema.py

```diff
@@ -96,6 +96,13 @@ class Query(graphene.ObjectType):
         Devices,
         description="Query that returns a list of all devices")
 
+    get_specific_device = graphene.Field(
+        Devices,
+        device_id=graphene.Int(),
+        description="Returns devices in a specific room using room id and accepts the argument\
+            \n- device_id: A unique identifier of the room"
+    )
+
```

Comment:
Avoid using verbs as variable name. Kindly rename `get_specific_device` to a noun variable